numpy/lib/tests/test_function_base.py

```diff
@@ -2688,6 +2692,63 @@ def test_with_minlength_and_weights(self):
         w = np.array([0.2, 0.3, 0.5, 0.1, 0.2])
         y = np.bincount(x, w, 8)
         assert_array_equal(y, np.array([0, 0.2, 0.5, 0, 0.5, 0.1, 0, 0]))
+    
+    def test_with_out(self):
+        x = np.array([1, 5, 2, 4, 1])
+        out = np.ones(6, dtype=int)
+        y = np.bincount(x, out=out)
+        assert_array_equal(out, np.array([0, 2, 1, 0, 1, 1]))
+
+    def test_with_minlength_and_out(self):
+        x = np.array([1, 5, 2, 4, 1])
+        out = np.zeros(6, dtype=int)
+        np.bincount(x, minlength=2, out=out)
```

Comment:
Here too I would assign to `y` and check `y is out` -- good to check things are as expected.